### ISSUE TITLE
Referral Reward 

### ISSUE THREAD

**Issue by SOULJAHMANN** [stackup-dev/open-source-campaign#362]
**Name of Proposer**
SOULJAHMANN 

**Is your feature request related to a problem? Please describe.**
It isn't a problem but a way of improving and encouraging people to make Stackup more popular and widely accepted 

**Describe the solution you'd like**
I suggest every successful referral should be rewarded with at least $1 to propel stickies to invite more people to the programme after the new stackie has registered 

**Describe alternatives you've considered**
After the referred stackie has performed a quest and approved, the stackie who referred such fellow should be rewarded with $1

**Additional context**
Every stackie should be rewarded and it will boost the community and increase competitiveness

**Comment by raybnsr** [comment]
I agree with this statement because using the concept of inviting others to use a referral code will make someone who refers someone will be excited if there are benefits that we get even in the form of a few points or $1 money. This will also increase StackUp to be more popular and known to many people.
This is also according to my personal experience which has succeeded in inviting several friends to enter but I haven't gotten any benefits. So if someone who manages to refer will benefit, maybe more people will be invited.

**Comment by otayiz** [comment]
I agree with this referral reward proposal. How about instead of a fixed ($1 or 1000 XP or whatever), referrers are rewarded based on activity of the people referred? Like referrer gets %10 of $'s or XP the person referred earns or something like that. 
That way people would have more incentive to refer others who actually use the platform to learn and earn. It can also be a way to avoid scammers-spammers-bots. 

Even at a $1 reward for referral, I'm sure someone somewhere would try to refer 1000 fake accounts to simply take out some money.

**Comment by Fluffyurbae** [comment]
I agree with this statement because by inviting others to use the referral code that we have, it will make us more excited if there is an advantage that we get even if it is in the form of a few points or $1 money. This will also increase StackUp to be known by many people.

**Comment by tegarmantanqorii** [comment]
I agree with this statement that there is a reward for users who share their reveral code, so that users are more active and creative in promoting the stackup. That will be a positive impact for stackup to be better known by many people with little capital.
